CONTRIBUTING.md

```diff
@@ -70,46 +101,51 @@ Basic rules:
 
 ## 📜 Pull Request Policy
 
-### Before landing
+### Before merging
+
+* Pull Requests must be open for at least 48 hours unless changes include errata fixes, infrastructure maintenance, or tests.
+* There must be no objections after 48 hours.
+* Tests must be included in Pull Requests for new features or bug fixes. If any test(s) are failing, you are responsible for fixing them.
 
-* Pull Requests must be open for at least 48 hours unless changes include errata fixes, infrastructure maintenance, or tests
-* There must be no objections after a 48 hour period
-* Tests must be included in Pull Requests for new features or bug fixes
+Each contribution is accepted only if there is no objection to it by a collaborator. During the review, collaborators may request that a specific contributor who is an expert in a particular area give an "LGTM" before the PR can be merged.
 
-The default for each contribution is that it is accepted once no collaborator has an objection. During review collaborators may also request that a specific contributor who is most versed in a particular area gives a "LGTM" before the PR can be merged.
+In the case that an objection is raised in a pull request by another collaborator, all collaborators involved should try to arrive at a consensus by addressing the concerns through discussion, compromise, or withdrawal of the proposed change(s).
 
-In the case of an objection being raised in a pull request by another collaborator, all involved collaborators should seek to arrive at a consensus by way of addressing concerns being expressed by discussion, compromise on the proposed change, or withdrawal of the proposed change.
+### When merging
 
-## When Landing
+* Do not use the **merge** option
```

Comment:
This can also go away. The only option that exists now is squash and merge. Rebasing and merge and a merge commit are disabled in this repo now.

Reply:
```suggestion
```